esmvaltool/utils/recipe_test_workflow/doc/source/user_guide/adding_a_recipe.rst

```diff
@@ -0,0 +1,87 @@
+Adding a recipe to the Recipe Test Workflow (|RTW|)
+===================================================
+
+.. include:: ../common.txt
+
+**Please note**: Before you follow these steps to add your recipe, you must be
+able to successfully run it with the latest version of ESMValTool on the
+compute server you use at your site (you should be able to find this in your
+site specific ``.cylc`` file in the site directory).
+
+#. Stop any running recipe_test_workflow workflows::
+
+    cylc stop "a_running_recipe_test_workflow"
+
+#. Locate your local ``<site>.cylc`` file, found in (recipe_test_workflow/site)
+   .
+
+#. Open it in your preferred code editor.
+
+#. Locate the ``[[COMPUTE]]`` section, it should look something like this::
+
+    [[COMPUTE]]
+    platform = <your_platform_here>
+    execution time limit = PT2M
+    [[[directives]]]
+    --wckey = RTW
+    --ntasks = {{ MAX_PARALLEL_TASKS }}
+    --mem = 2G
+
+#. Run your recipe with ESMValTool on your compute server for your site.
+
+#. If the recipe takes less than 10 minutes to run then it should be added as a
+   "fast" recipe in the ``flow.cylc`` file within the ``[task parameters]``
+   section.If it takes longer than ten minutes it should be included in "medium
+   ".
+
+#. If either of the memory readings from your run are larger than the
+   values specified in the ``[[COMPUTE]]`` section, you need to add your recipe as
+   another `process` similar to::
+
+    [[process<fast=recipe_albedolandcover>]]
+    # Actual: 0m31s, 2.5 GB on 2024-04-08.
+    execution time limit = PT2M
+    [[[directives]]]
+        --mem = 3G
+
```

Comment:
Add a note here to say to round to the nearest second in the comment (i.e. there is no need to copy the full duration from the messages printed to screen), see https://github.com/ESMValGroup/ESMValTool/pull/3627#discussion_r1618743351.